apps/api/src/app/events/usecases/parse-event-request/parse-event-request.usecase.ts

```diff
@@ -104,18 +104,12 @@ export class ParseEventRequest {
 
     command.payload = merge({}, defaultPayload, command.payload);
 
-    let jobData = {
+    const jobData = {
       ...command,
-      actor: command.actor,
       transactionId,
     } as ParseEventRequestCommand;
 
-    if ((command as ParseEventRequestMulticastCommand).to?.length > 0) {
-      jobData = jobData as ParseEventRequestMulticastCommand;
-      jobData.to = (command as ParseEventRequestMulticastCommand).to;
```

Comment:
Done by me this month, it is not needed as it will be destructured above in `...command,`.
More than that i regret that i created two types of one usecase command, Apologies in advance, i will revisit it in order to think of a better way.